14/umbraco-cms/tutorials/creating-a-backoffice-api/polymorphic-output-in-the-management-api.md

```diff
@@ -0,0 +1,113 @@
+﻿---
+description: How to support polymorphic outputs from custom Management APIs
+---
+
+# Polymorphic output in the Management API
+
+For security reasons, the `System.Text.Json` serializer will not serialize types that are not explicitly referenced at compile time.
+
+This can be a challenge with when dealing with polymorphic API outputs. As a workaround, the Management API provides two options for enabling polymorphic outputs.
```

Comment:
```suggestion
This can be a challenge when dealing with polymorphic API outputs. As a workaround, the Management API provides two options for enabling polymorphic outputs.
```

Reply:
👍